apps/angular/59-content-projection-defer/src/app/expandable-card.ts

```diff
@@ -41,14 +66,29 @@ import { ChangeDetectionStrategy, Component, signal } from '@angular/core';
       class="overflow-hidden transition-[max-height] duration-500"
       [class.max-h-0]="!isExpanded()"
       [class.max-h-[1000px]]="isExpanded()">
-      <ng-content />
+      @defer (on interaction(expandContentButton)) {
```

Comment:
yes, yes and yes 
I was looking for this solution. There were a hint in the title. But I think I will create a second challenge for people to guess this solution. 

Great job. 🔥